dialogflow-app.js

```diff
@@ -1000,6 +1009,7 @@ class DialogflowApp extends AssistantApp {
     } : {
       richResponse: textToSpeech
     });
+    console.log('data:google', google);
```

Comment:
We've been using `debug(...)` or `error(...)` for printing messages. Though I'm also not sure that we want this print at all.

Reply:
yeah, my error :( I forget about this little helper